docker-compose.yml

```diff
@@ -0,0 +1,138 @@
+version: '3.5'
+
+volumes:
+  db_socket:
+
+services:
+  composer:
+    image: composer
+    working_dir: /app
+    volumes:
+      - .:/app
+    entrypoint:
+      - sh
+      - -c
+      - |
+        exec tail -f /dev/null
+
+  php53:
```

Comment:
Instead of multiple service, we could merge together multiple docker-compose files
and with a Makefile choose which PHP container to start:
something like `docker-compose --file=docker-compose.yml --file=docker/docker-compose.php5.yml`

WDYT?

Reply:
Nice refacto.

I do not have time to implements right now.